src/Microsoft.ML.EntryPoints/CrossValidationMacro.cs

```diff
@@ -95,7 +95,7 @@ public sealed class Output
         {
             [TlcModule.Output(Desc = "The final model including the trained predictor model and the model from the transforms, " +
                 "provided as the Input.TransformModel.", SortOrder = 1)]
-            public PredictorModel[] PredictorModel;
+            public PredictorModel[] PredictorModels;
```

Comment:
>PredictorModels [](start = 36, length = 15)

i think this might have side-effects in the entry points; but that is probably ok. 

@yaeldekel might have an opinion too.

Reply:
You're right, it did break some entry point tests. I realize this one is not an `ArgumentAttribute` to begin with, so I'm reverting it in my next change (I haven't pushed yet).

Many other entry point tests also broke because of renaming `Column` to `Columns` and setting `Name = "Column"`. Basically, the graph runner ctor checks for validity of node names, but it only checks against `ArgumentAttribute.Name` and the `ArgumentAttribute.Aliases`, and since `Name` was set to "Column" and Aliases were the `ShortName` list, all of those checks failed. 

It wouldn't have been correct to change `ShortName` to "col,Columns` as that is visible on the command line, and "Columns" doesn't make sense on the command line. I ended up modifying the check in `InputBuilder`.